### PR TITLE
Add feedback cmd

### PR DESCRIPTION
**Pull request by madebyrogal**
## Description
Changes proposed in this pull request:
- Add feedback command
- Add feedback command to help message

## Related issue(s)

- #681

### CODE REVIEW

File: pkg/bot/interactive/help.go [hunk 1]
```diff
@@ -81,6 +82,14 @@ func Help(platform config.CommPlatformIntegration, clusterName, botName string)
 					},
 				},
 			},
+			{
+				Base: Base{
+					Description: "Angry? Amazed?",
+				},
+				Buttons: []Button{
+					btnBuilder.DescriptionURL("Give feedback", "feedback", "https://feedback.botkube_io", slack.StylePrimary),
```
Comment (mszostok) [inline pkg/bot/interactive/help.go:90]:
```suggestion
					btnBuilder.DescriptionURL("Give feedback", "feedback", "https://feedback.botkube.io", slack.StylePrimary),
```

please also fix it for twitter in line: 97. Thanks!

**Review by mszostok — COMMENTED** [review]
Two comments, besides that, LGTM 🚀

File: pkg/bot/interactive/feedback.go [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+	return Message{
+		Sections: []Section{
+			{
+				Base: Base{
+					Body: Body{
+						Plaintext: ":wave: Hey, what's your experience with Botkube so far?",
+					},
+				},
+				Buttons: []Button{
+					btnBuilder.ForURL("Give feedback", "https://feedback.botkube.io", slack.StylePrimary),
```
[the remark this answers is not in the document]
Comment (madebyrogal) [reply pkg/bot/interactive/feedback.go:17]: Yes completely agree I was so focus on delivering functionality that I forget about it
Only one small improvement in `convertToSlackStyle ` by default it should return slack.StyleDefault

File: pkg/bot/slack.go [hunk 3]
```diff
@@ -260,7 +262,16 @@ func (b *Slack) handleMessage(event slackMessage) error {
...
-	e := b.executorFactory.NewDefault(b.commGroupName, b.IntegrationName(), b, isAuthChannel, info.Name, channel.Bindings.Executors, request)
+	e := b.executorFactory.NewDefault(execute.NewDefaultInput{
+		CommGroupName:   b.commGroupName,
+		Platform:        b.IntegrationName(),
+		NotifierHandler: b,
+		IsAuthChannel:   isAuthChannel,
+		ConversationID:  info.Name,
+		Bindings:        channel.Bindings.Executors,
+		Message:         request,
+		User:            event.User,
```
[the remark this answers is not in the document]
Comment (madebyrogal) [reply pkg/bot/slack.go:273]: It's used in slack PostEphemeral (only visible for you) pkg/bot/slack.go:318 in send function

**Review by mszostok — APPROVED** [review]
LGTM 🚀

**Pull request merged**